tests/e2e/utils/deactivate-utility-plugins.js

```diff
@@ -5,30 +5,30 @@
 import { switchUserToAdmin, visitAdminPage, switchUserToTest, isCurrentURL } from '@wordpress/e2e-test-utils';
 
 /**
- * Deactivate all plugins except Site Kit.
+ * Deactivate Site Kit utility plugins.
  */
-export async function deactivateAllOtherPlugins() {
+export async function deactivateUtilityPlugins() {
```

Comment:
One note, actually: maybe call this `deactivateTestPlugins` or `deactivateTestUtilityPlugins`? "Utility" is a bit vague and doesn't make clear to a casual observer what the purpose of these plugins is or what they're used for.

Labelling them explicitly as plugins used during testing would clarify this.